src/accounts/urls.py

```diff
@@ -30,4 +31,9 @@
          name='contributor_profile'),
 
     path(_('logout/'), auth_views.LogoutView.as_view(), name='logout'),
+
+    path(_('register-newsletter/'), NewsletterView.as_view(),
+         name='register-newsletter'),
+    path(_('newsletter-success/'), NewsletterSuccessView.as_view(),
```

Comment:
```suggestion
    path(_('register-newsletter-success/'), NewsletterSuccessView.as_view(),
```

pour être semblable au workflow register existant